modules/social_features/social_post/social_post.install

```diff
@@ -4,9 +4,6 @@
  * Install, update and uninstall functions for the social_comment module.
  */
 
-use Drupal\Core\Database\Database;
-use Symfony\Component\Yaml\Yaml;
-
 /**
```

Comment:
Perhaps add some information here that you should not add update hooks in this file. Your future self will thank you!